### DISCUSSION TITLE
Welcome to Our Community Discussions! | 欢迎来到我们的社区讨论区！

### DISCUSSION THREAD

**Discussion by FoxSensei001** [FoxSensei001/LoveIwara#3] [Announcements]
# Welcome to Our Community Discussions! | 欢迎来到我们的社区讨论区！

👋 Hello everyone! We're excited to announce that we've opened GitHub Discussions for our community! 
大家好！我们很高兴地宣布，我们已经开放了 GitHub Discussions 社区讨论功能！

## What is Discussions? | 什么是 Discussions？
Discussions is a collaborative communication forum for our community. Here you can:
Discussions 是一个供社区成员协作交流的平台。在这里你可以：

- Ask questions and share ideas
- Engage with other community members
- Share your experiences and best practices
- Discuss feature requests and project direction

- 提出问题和分享想法
- 与其他社区成员互动
- 分享你的经验和最佳实践
- 讨论功能需求和项目方向

## Categories | 分类
- 📢 Announcements: Official project updates and announcements
- 💭 General: For general discussions and conversations
- 💡 Ideas: Share your suggestions for the project
- 📊 Polls: Vote and help make community decisions
- 🙏 Q&A: Ask questions and get help from the community
- 🎨 Show and tell: Share what you've built or learned

- 📢 公告：官方项目更新和通知
- 💭 综合讨论：用于一般性讨论和交流
- 💡 想法：分享你对项目的建议
- 📊 投票：参与投票，协助社区决策
- 🙏 问答：向社区寻求帮助和提问
- 🎨 展示与分享：分享你的成果和心得

## Prohibited Content | 禁止内容
- No NSFW (Not Safe For Work) content
- No political content/discussions 
- No hate speech or harassment
- No spam or advertisements
- No personal information

- 禁止发布成人/不当内容
- 禁止政治相关内容/讨论
- 禁止仇恨言论或骚扰行为
- 禁止垃圾信息或广告
- 禁止个人隐私信息

## Guidelines | 指南
Please remember to:
请记住：

- Be respectful and kind to others
- Stay on topic
- Follow our code of conduct
- Search before asking questions
- Use clear and descriptive titles

- 尊重他人，保持友善
- 保持话题相关性
- 遵守行为准则
- 提问前先搜索是否已有相关讨论
- 使用清晰描述性的标题

We're looking forward to building an active and engaging community together!
我们期待与大家一起建立一个活跃且充满互动的社区！

Happy discussing! 🎉
祝讨论愉快！🎉

**Comment by lyc736068585** [comment]
作者大大你好，为啥在不全屏的情况下，视频画面占比这么小？能否调整一下非全屏情况下画面的占比问题
主要感觉不至于每打开一个视频，都要使用全屏看吧

![1](https://github.com/user-attachments/assets/06a40e74-8e66-4188-b61a-23a75f6aa55d)

**Reply by FoxSensei001** [reply]
嗯，这里后续会优化下。

**Reply by jiaojie1234** [reply]
你好，我想问一下网络设置那个代理要咋么填写啊

**Comment by 2080604952** [comment]
![efecb1da-d9d8-4235-a706-7759998af07b](https://github.com/user-attachments/assets/4637f509-a1c2-42c8-8579-a0048ef69026)
大佬为什么我这里一直加载失败外加。登录不上去呢？

**Reply by FoxSensei001** [reply]
去设置-网络设置里配置上代理

**Reply by 2080604952** [reply]
> 去设置-网络设置里配置上代理

![image](https://github.com/user-attachments/assets/710265ea-26a1-44c7-8104-e6f52589ebed)
佬为什么提示代理错误

**Reply by FoxSensei001** [reply]
不清楚。理论上，如果你填的就是梯子暴露的http或混合端口，且节点可用，就不会有这种问题。

**Reply by 2080604952** [reply]
> 不清楚。理论上，如果你填的就是梯子暴露的http或混合端口，且节点可用，就不会有这种问题。
这次又成别的错误了
![Uploading 02ba4000-1cbc-4961-b02e-5cfdbf913da3.png…]()

**Comment by 2080604952** [comment]
是不是因为flutter没有搭建好环境？



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2025年3月22日(周六) 中午1:52
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [FoxSensei001/LoveIwara] Welcome to Our Community Discussions! | 欢迎来到我们的社区讨论区！ (Discussion #3)




 
不清楚。理论上，如果你填的就是梯子暴露的http或混合端口，且节点可用，就不会有这种问题。

**Reply by FoxSensei001** [reply]
你可以试试用clash，我自己用的就是clash verge，这里填的 127.0.0.1:混合端口。

**Reply by 2080604952** [reply]
> 你可以试试用clash，我自己用的就是clash verge，这里填的 127.0.0.1:混合端口。

用的是clash啊
![1e9f5812-e2a2-4e73-bc0a-a4a3fdd272e7](https://github.com/user-attachments/assets/d90ac53a-0564-4b7b-8a35-03ce9873ffbf)

**Reply by FoxSensei001** [reply]
clash - 设置 - 端口设置，右侧显示的就是混合代理端口号，请确保此处和应用内填写的一致

**Reply by 2080604952** [reply]
> clash - 设置 - 端口设置，右侧显示的就是混合代理端口号，请确保此处和应用内填写的一致

感谢

**Comment by APPCDQ** [comment]
![Screenshot_20250709_211037](https://github.com/user-attachments/assets/8560ad5c-860a-459c-a287-b7cb78eb0441)
![Screenshot_20250709_211039](https://github.com/user-attachments/assets/84b55c0e-e597-4d58-bb8b-5a6550a5ce76)
提示代理正常工作，但是仍无法使用

**Reply by WStarssXing** [reply]
有可能被服务器拉黑了这个节点

**Comment by heartsperch** [comment]
如果我自己想做一个类似的应用的话去哪里查api信息？

**Reply by FoxSensei001** [reply]
以 http 请求为例，如果请求来源于网页，你可以通过 "F12 控制台 - Network" 非常轻松的找到当前网页触发的 API 请求，对于应用则需要有 WEB 抓包工具通过拦截请求获取内容。当然获取 api 的途径还有去翻阅网络上别人分享的技术文档、项目或者官方直接提供的文档等方式，看具体情况。
有些 API 是直接以 JSON 格式返回的数据信息，有些则是 SSR 服务端渲染得到的 HTML 文本，前者用于前后端分离的应用场景，会直接提供给你有用的信息，后者则需要通过 css 语法对 html 进行解析以提取你想要的内容。
有时候应用为了防止API 被第三方调用，会有 Cloudflare 盾的保护，需要你懂一些过盾方面的知识，还有一些是通过混淆后的代码根据特定的规则生成 请求密钥 放在请求头里才允许你访问接口，所以可能还需要你有一定的爬虫知识。

**Comment by heartsperch** [comment]
好的，感谢分享



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2025年08月31日 12:22 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [FoxSensei001/LoveIwara] Welcome to Our Community Discussions! | 欢迎来到我们的社区讨论区！ (Discussion #3) |

以 http 请求为例，如果请求来源于网页，你可以通过 "F12 控制台 - Network" 非常轻松的找到当前网页触发的 API 请求，对于应用则需要有 WEB 抓包工具通过拦截请求获取内容。当然获取 api 的途径还有去翻阅网络上别人分享的技术文档、项目或者官方直接提供的文档等方式，看具体情况。
有些 API 是直接以 JSON 格式返回的数据信息，有些则是 SSR 服务端渲染得到的 HTML 文本，前者用于前后端分离的应用场景，会直接提供给你有用的信息，后者则需要通过 css 语法对 html 进行解析以提取你想要的内容。
有时候应用为了防止API 被第三方调用，会有 Cloudflare 盾的保护，需要你懂一些过盾方面的知识，还有一些是通过混淆后的代码根据特定的规则生成 请求密钥 放在请求头里才允许你访问接口，所以可能还需要你有一定的爬虫知识。

**Reply by jiaojie1234** [reply]
你好，安卓手机如何在网络设置那里配置代理

**Comment by rancanyu19640411-beep** [comment]
![Screenshot_20250908_011800](https://github.com/user-attachments/assets/bdf8d54f-6bd4-4e6c-9b0c-99d4851c05ed)
无法播放

**Reply by FoxSensei001** [reply]
自身网络问题

**Comment by jiaojie1234** [comment]
你好作者，我不会用网络设置的配置代理，可以教一下吗，谢谢

**Reply by jiaojie1234** [reply]
手机和电脑都不会使用配置代理

**Comment by arotamiri** [comment]
![Screenshot_20260411_015649.jpg](https://github.com/user-attachments/assets/7f9abf05-3c53-485e-b1c5-bd27428b4127)



开发者大大您好！在邮箱有效、验证码输入正确的情况下，无法注册、反复提示ValidationError是什么原因呢？
